badCode.rb

```diff
@@ -0,0 +1,5 @@
+def badName
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Naming/MethodName: Use snake_case for method names.